### DISCUSSION TITLE
Some app icons keep changing back to the original

### DISCUSSION THREAD

**Discussion by atticusmatticus** [elrumo/macOS_Big_Sur_icons_replacements#1465] [General]
The title says it all. Some apps seem to replace or set their icons back to the originals upon startup/shutdown. The Battle.net app from Blizzard Entertainment is an example of this. I'll change the app icon via the usual method of dragging it to the get info box. Then I'll launch the app and it will have the new icon in the dock as desired. Then if I quit the app and relaunch it, it will have the old original icon. Does anyone know how to fix this? Thanks.

**Comment by Konrad-Waldhe** [comment]
Maybe a little late but, right click the app in finder  > "show contents" > ⌘ + ⇧ + . (to show hidden files) > right click "Icon?" > "Get info" > tick "locked" checkbox.

**Comment by vikasbhatia** [comment]
I use [Pictogram](https://pictogramapp.com) to change app icons. It automatically reapplies them across app updates/when the app tries to change it back to stock. I don't have any experience with Battle.net, but maybe it'll help you.

**Comment by atticusmatticus** [comment]
Thanks for the suggestions! I'll give them a try.
